test/images/iperf/BASEIMAGE

```diff
@@ -0,0 +1,4 @@
+amd64=gcr.io/google_containers/ubuntu-slim:0.12
```

Comment:
@ixdy Would `debian-base` be a better choice than `ubuntu-slim` in the cases where both work?